### ISSUE TITLE
No playlists are seen on any YT channel

### ISSUE THREAD

**Issue by AYakovleff** [polymorphicshade/Tubular#182]
### Checklist

- [X] I am able to reproduce the bug with the [latest version](https://github.com/polymorphicshade/Tubular/releases/latest).
- [X] I am *not* able to reproduce the bug with the *latest version* of [vanilla NewPipe](https://github.com/TeamNewPipe/NewPipe/releases/).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/polymorphicshade/Tubular/issues) or [closed](https://github.com/polymorphicshade/Tubular/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.
- [X] I have read and understood the vanilla NewPipe [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Affected version

0.27.2

### Steps to reproduce the bug

Playlists on any YT channel are not shown in Tubular.

### Expected behavior

Playlists should be seen and clickable and playable.

### Actual behavior

No playlists are seen.

### Screenshots/Recordings

![Screenshot_20241206-225233_Tubular](https://github.com/user-attachments/assets/75656169-25c2-4b00-9295-d8407e194565)
![Screenshot_20241206-225320_Brave](https://github.com/user-attachments/assets/e75c6d46-17fa-4fdc-9c77-bf54785839e3)
![Screenshot_20241206-225337_Tubular](https://github.com/user-attachments/assets/023a89aa-98c4-48dd-8ea1-cb651d6500e2)
![Screenshot_20241206-225356_Brave](https://github.com/user-attachments/assets/bf5a6173-b98f-4b65-b77d-38a990601a72)

### Affected device model

Samsung A40, Samsung tab a 10.1

**Comment by brnwlshubh** [comment]
@christian179717 @AYakovleff It occurs due to lack of Update.
It will work post update.

**Comment by sugamd53** [comment]
Same problem 

![Screenshot_2024-12-16-09-13-57-313_org.polymorphicshade.tubular.jpg](https://github.com/user-attachments/assets/b57ee795-3379-4c4a-8ab8-1ca74bc777ed)

**Comment by aloi5862** [comment]
i think the project / account is abandoned by the owner. @polymorphicshade don't contribute to any project after november and since july became less active on github. you can see here https://github.com/polymorphicshade.

**Comment by justeepin** [comment]
Of course. Of all the alternate YouTube apps, Tubular is my favorite. Oh well, time to search again.

**Comment by christian179717** [comment]
@justeepin could you please give an update when you've found an alternative?
I have grown fond of tubular over the last few months. I don't want to lose functions such as shuffle and play or the integration of Sponsor Block.

**Comment by justeepin** [comment]
I've actually just given up for now and gone back to YouTube.

**Comment by asandikci** [comment]
Probably solved with latest release v0.27.5, request re-open if issue persist

**Comment by christian179717** [comment]
@justeepin Tubular works now again. I switched temporary to this one: https://github.com/mikooomich/Tubular

**Comment by justeepin** [comment]
Thanks. I realized one other issue though... I can't stream it to my TV and there isn't a Tubular for Roku.
